docs/faq/troubleshooting/why-did-codacy-stop-commenting-on-pull-requests.md

```diff
@@ -14,7 +14,15 @@ If you stopped receiving coverage summaries on your pull requests, please [revie
 
 Different reasons can cause Codacy to stop analyzing and commenting on pull requests, but the most common is that the user who initially enabled the GitLab or Bitbucket integration no longer has permissions on the repository or that the SSH key is no longer valid.
 
-To fix this issue and avoid future disruptions, refresh the GitLab or Bitbucket integration on Codacy using a dedicated service account on your Git provider:
+To fix this issue and avoid future disruptions, refresh the GitLab or Bitbucket integration on Codacy [using a dedicated service account](#service-account) on your Git provider.
+
+## Using a dedicated service account to integrate with your Git provider {: id="service-account"}
```

Comment:
@nicklem This is the option of isolating the service account procedure on a dedicated section within the same troubleshooting page.

However, I decided to try out a second option - isolating the procedure on [a dedicated page](https://docs-680-dedicated-service-account--docs-codacy.netlify.app/repositories-configure/integrations/using-a-dedicated-service-account/). 🤓 

The page title is a bit long, but it might work... Thoughts?

Reply:
I like the idea of the separate page, but for completeness, I'd also add one sentence to explain the concept of "service account" (e.g., I like [this explanation](https://www.silverfort.com/glossary/service-account/)). I didn't even know it had a name 😅